internal/driver/glfw/menu_bar_test.go

```diff
@@ -182,7 +182,7 @@ func TestMenuBar(t *testing.T) {
 				{
 					actions:    []action{{"tap", fileNewPos}},
 					wantAction: "new",
-					wantImage:  "menu_bar_initial.png",
+					wantImage:  "menu_bar_initial1.png",
```

Comment:
why postfixing numbers? it is not clear what the intended differences are

Reply:
This relates to my trying to fix the tests. It did not suceed. The problem is the color changes when a button is focused. Previously the buttons did not change color when focused, as it was not possible to focus them. So the menu_bar_initial.png can not be used in every case. It will be different depending on the previous simulated operations. Further changes to the tests must be done later on, introducing new png images. This must wait until the coloring sheme is settled.

Reply:
Please use appropriate naming instead of adding a number - it should be clear of the image content/state based on the name. We can’t have failing tests waiting for another PR or external change, if this PR depends on another then we have to land that first before this can complete.